packages/server/src/models/shelf.entity.ts

```diff
@@ -0,0 +1,28 @@
+import { Column, Entity, PrimaryGeneratedColumn, PrimaryColumn } from 'typeorm';
+
+@Entity()
+export class Shelf {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  name: string;
+
+  @Column()
+  description: string;
+
+  @Column()
+  location: string;
+
+  @Column()
+  pictureUrl: string;
```

Comment:
Shouldn't `Self` be connected to other tables somehow? Say we want to have simple UI for a given shelf with a number if flowers and some info about people connected to it.

Reply:
Fixed

Reply:
Fixed